### PR TITLE
[14.0] project_stage_closed: deprecated module

### PR DESCRIPTION
**Pull request by petrus-v**
This module is useless as long odoo added the is_closed field
in version 14.0.

@pedrobaeza as discussed earlier in https://github.com/OCA/project/pull/765 !

### CODE REVIEW

File: project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+def migrate(cr, _):
+    logger.info(
+        "Set project.task.type is_closed field using closed field if set "
+        " and drop closed field"
+    )
+    cr.execute(
+        """UPDATE project_task_type SET is_closed = closed
+            WHERE NOT closed is not NULL;
```
Comment (pedrobaeza) [inline project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py]:
```suggestion
            WHERE closed is not NULL;
```

File: project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import logging
```
Comment (pedrobaeza) [inline project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py]: No need to over-log. Odoo already mentions about executing migration script, but I let it to your taste.

Comment (pedrobaeza) [inline project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py]: This should be done on pre, not on post, as on post the column is already dropped by Odoo update mechanism.

File: project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+def migrate(cr, _):
+    logger.info(
+        "Set project.task.type is_closed field using closed field if set "
+        " and drop closed field"
+    )
+    cr.execute(
+        """UPDATE project_task_type SET is_closed = closed
+            WHERE NOT closed is not NULL;
+        """
+        "ALTER TABLE project_task_type DROP COLUMN closed;"
```
Comment (pedrobaeza) [inline project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py]: Don't remove the column. Odoo already does it.

File: project_stage_closed/models/__init__.py [hunk 4]
```diff
@@ -1,2 +0,0 @@
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (pedrobaeza) [inline project_stage_closed/models/__init__.py:1]: I think you should strip from main `__init__.py` the `from . import models` clause.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py (hunk 1 above)
Comment (petrus-v) [reply project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py]: oups :eyes:

File: project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py (hunk 3 above)
Comment (petrus-v) [reply project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py]: I've been tricked by odoo message

> 2021-05-23 18:52:48,702 30278 WARNING oca-project odoo.addons.base.models.ir_model: Deleting field project.task.type.closed (hint: fields should be explicitly removed by an upgrade script) 

Thanks

File: project_stage_closed/models/__init__.py (hunk 4 above)
Comment (petrus-v) [reply project_stage_closed/models/__init__.py:1]: I've remove folder and init file

**Review by pedrobaeza — APPROVED** [review]
Thanks!

**Pull request merged**